### PR TITLE
Feature/my account

### PR DESCRIPTION
**Pull request by KamikazeBVB**

### CODE REVIEW

File: src/universal/containers/app/MyAccountPage/MyAccountPage.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+} from './../../../components';
+import {initialize} from 'redux-form';
+import { connect } from 'react-redux';
+import { updatePersonalInformationAction } from './../../../redux/reducers/auth';
+/*
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { initialize } from 'redux-form';
+import { pushState } from 'redux-router';
+*/
```
Comment (liviuignat) [inline src/universal/containers/app/MyAccountPage/MyAccountPage.js]: I think you forgot some comments here.

File: src/universal/containers/app/MyAccountPage/MyAccountPage.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  componentWillMount() {
+    const {
+      user: {
+        firstName,
+        lastName,
+        companyName
+      }
+    } = this.props;
+
+    this.props.initialize('PersonalInformationForm', {firstName, lastName, companyName});
```
Comment (liviuignat) [inline src/universal/containers/app/MyAccountPage/MyAccountPage.js]: Put the string `'PersonalInformationForm'` inside `PersonalInformationForm` component.

PersonalInformationForm:
```
export const PERSONAL_INFORMATION_FORM_NAME = 'PersonalInformationForm';
```
MyAccountPage:
```
import PersonalInformationForm, {PERSONAL_INFORMATION_FORM_NAME} from './PersonalInformationForm';
....
this.props.initialize(PERSONAL_INFORMATION_FORM_NAME, {firstName, lastName, companyName});
```

File: src/universal/containers/app/MyAccountPage/MyAccountPage.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import "../../../theme/variables.scss";
+
+.MyAccountPage {
+  width: 600px;
+  margin: 0 auto;
```
Comment (liviuignat) [inline src/universal/containers/app/MyAccountPage/MyAccountPage.scss]: I'm not really sure if the form should be middle aligned, let's see how it looks left aligned and remove this line :).

File: src/universal/containers/app/MyAccountPage/PersonalInformationForm.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import {reduxForm} from 'redux-form';
+import { FormTextField, RaisedButton } from './../../../components';
+
+@reduxForm({
+  form: 'PersonalInformationForm',
```
Comment (liviuignat) [inline src/universal/containers/app/MyAccountPage/PersonalInformationForm.js]: Declare and use the `const PERSONAL_INFORMATION_FORM_NAME ` also here.

File: src/universal/containers/app/MyAccountPage/PersonalInformationForm.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+    return (
+      <div>
+        <form onSubmit={handleSubmit}>
+          <div>
+            <FormTextField field={firstName}
+              type="text"
+              labelText="First name"
+              fullWidth
+              disabled={isSaving}/>
```
Comment (liviuignat) [inline src/universal/containers/app/MyAccountPage/PersonalInformationForm.js]: `isSaving` is set to false? `isSaving` should be set from the reducer and handled here

File: src/universal/redux/reducers/auth/__tests__/authReducer-test.js [hunk 6]
```diff
@@ -5,10 +5,142 @@ import * as actions from './../actions';
...
+    it('SHOULD have the error set to the correct message',
+      () => expect(currrentState.updateUserError).to.equal(updateUser.error));
+  });
+
+  describe('WHEN updateting the user details is succesfull', () => {
+    const updateUser = {
+      type: actions.UPDATE_CURRENT_USER_SUCCESS,
+      result: {
+        email: 'liviu@ignat.email',
+        password: 'blabla',
```
Comment (liviuignat) [inline src/universal/redux/reducers/auth/__tests__/authReducer-test.js]: So this is Bill Gates password? :D

File: src/universal/redux/reducers/auth/__tests__/authReducer-test.js [hunk 7]
```diff
@@ -5,10 +5,142 @@ import * as actions from './../actions';
...
+        }
+      };
+
+      beforeEach(() => currrentState = reducer(currrentState, updateUserApiToken));
+
+      it('SHOULD update the token correctly',
+        () => expect(currrentState.user.apiAccessToken).to.equal(updateUserApiToken.result.apiAccessToken));
+
+      it('SHOULD have the update in progress flag set to false',
+      () => expect(currrentState.updatetingUserAPIAccessToken).to.equal(false));
```
Comment (liviuignat) [inline src/universal/redux/reducers/auth/__tests__/authReducer-test.js]: extra spaces required in front of `()`

File: src/universal/containers/app/MyAccountPage/MyAccountPage.js (hunk 1 above)
Comment (KamikazeBVB) [inline src/universal/containers/app/MyAccountPage/MyAccountPage.js]: Fixed.

**Pull request merged**